clients/_test/clientSearchRechercheEntreprise/_snapshots/search-800329849.json

```diff
@@ -32,7 +32,7 @@
           "commune": "PARIS 13",
           "activitePrincipale": "90.01Z",
           "libelleActivitePrincipale": "Arts du spectacle vivant (90.01Z)",
-          "trancheEffectif": "02",
+          "trancheEffectif": "N",
```

Comment:
@HAEKADI this is odd. This is our test snapshot and it tells `trancheEffectif` changed and company became `Unite non employeuse`. 

Yet I do not find the same result on sirene.fr 800329849

Could it be a bug on sirene API ? Or a bug on our side ?

Reply:
I am confused. Both the api and sirene.fr indicate the same thing. What snapshot are we talking about ?

Reply:
Ok thie issue might lay in  another variable : `caractereEmployeur` is set to `N` for these entities

Reply:
Ok I dont understand, so I will write a mail to Insee team

Reply:
Insee team answered. 
Ths `caractereEmployeur` and `trancheEffectif` come from two differents sources.
Either one can be the most up to date and we currently dont know which is.